packages/apify/src/crawlers/cheerio_crawler.ts

```diff
@@ -335,6 +335,7 @@ export interface CheerioHandlePageInputs extends CrawlingContext {
      */
     body: (string | Buffer);
 
+    // TODO: maybe consider making this `unknown` instead to force casting for TS users
```

Comment:
why not make it another generic argument (and default to unknown)

Reply:
Can do that, sure